scipy/stats/_continuous_distns.py

```diff
@@ -781,12 +787,39 @@ def _pdf(self, x, c, d):
         # burr.pdf(x, c, d) = c * d * x**(-c-1) * (1+x**(-c))**(-d-1)
         return c * d * (x**(-c - 1.0)) * ((1 + x**(-c))**(-d - 1.0))
 
+    def _logpdf(self, x, c, d):
+        return np.log(c) + np.log(d) + (-c - 1.0) * np.log(x) \
+                + (-d - 1.0) * np.log1p(x**(-c))
+
     def _cdf(self, x, c, d):
         return (1 + x**(-c))**(-d)
 
+    def _logcdf(self, x, c, d):
+        return -d * np.log1p(x**(-c))
+
+    def _sf(self, x, c, d):
+        return 1 - (1 + x**(-c))**(-d)
+
     def _ppf(self, q, c, d):
         return (q**(-1.0/d) - 1)**(-1.0/c)
 
+    def _stats(self, c, d):
+        nc = np.arange(1, 5)/c
+        e1, e2, e3, e4 = sc.beta(d + nc, 1. - nc) / sc.beta(d, 1.0)
```

Comment:
Simplify: `beta(d, 1.0)` is just `1/d`, so this expression can be written `d * sc.beta(d + nc, 1. - nc)`.

Reply:
right, same as `_munp`, done.